docs/user-manuals/memory-evict.md

```diff
@@ -0,0 +1,130 @@
+# Eviction Strategy base on Memory Usage
+
+## Introduction
+
+Koordinator supports the dynamic overcommitment from idle resources on node to low-priority
+Pods as Batch priority. In co-location scenarios, the actual memory resource usage of 
+nodes is constantly changing. For incompressible resources such as memory, high resource 
+usage of node may cause OOM, which results in the high-priority Pod got killed. Koordiantor 
+provides an eviction strategy based on the memory usage node. `Koordlet` will continuously 
+detect the memory usage of node (Total-Available) in second-level granularity. 
+When the resource memory usage of node is high, it will evict low-priority BE Pods to 
+ensure the QoS of high-priority pods until the memory usage of node reduces below to the 
+threshold (evictThreshold). During the eviction process, Pods with lower priority(Pod.Spec.Priority)
+will be selected first, and if the priority is the same, Pods which consume more memory will be 
+evicted first.
+
+
+![image](/img/memory-evict.svg)
+
+### Prerequisite
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to 
+[Installation](/docs/installation).
+
+| Component | Version Requirement |
+| --- | ------- |
+| Kubernetes | ≥v1.18 |
+| koordinator | ≥v0.3.0 |
+
+The eviction strategy is provided by `Koordlet`, which is disabled by default in feature-gate.
+Please make sure the `BEMemoryEvict=true` field has been added in the `-feature-gates` arguments of `Koordlet`
+as the [example](https://github.com/koordinator-sh/charts/blob/main/versions/v1.2.0/templates/koordlet.yaml#L36)。
+
+## Use Eviction Strategy base on Memory Usage
+
+1. Create a configmap.yaml file based on the following ConfigMap content:
+   ```yaml
+   #ConfigMap slo-controller-config example。
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: slo-controller-config # name should be set as the configuration of koord-manager, e.g. ack-slo-config 
```

Comment:
perhaps remove the naming like `ack-xxx`?

Reply:
for ack user better understanding